src/matter/cluster/LevelControlCluster.ts

```diff
@@ -0,0 +1,181 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { OptionalAttribute, WritableAttribute, OptionalWritableAttribute, Attribute, Cluster, Command, OptionalCommand, TlvNoResponse } from "./Cluster";
+import { BitFlag, MatterApplicationClusterSpecificationV1_0, TlvBitmap, TlvEnum, TlvField, TlvNullable, TlvObject, TlvUInt16, TlvUInt8 } from "@project-chip/matter.js";
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.6.5.8 */
+const optionsBitmap = TlvBitmap(TlvUInt8, {
+    /** Dependency on On/Off cluster. */
+    executeIfOff: BitFlag(0), // TODO default: true.
+
+    /** Level effect on color. */
+    coupleColorTempToLevel: BitFlag(1),
+});
+
+/** @see {@link MatterCoreSpecificationV1_0} § 1.6.6.1 */
+const MoveToLevelCommandRequest = TlvObject({
+    level: TlvField(0, TlvUInt8.bound({max:254})),
+    transitionTime: TlvField(1, TlvNullable(TlvUInt16)),
+    optionsMask: TlvField(2, optionsBitmap),
+    optionsOverride: TlvField(3, optionsBitmap), // TODO: 0 Default - all optionsOverride
```

Comment:
I think default 0 should work  to set

Reply:
Maybe it's my lack of Typescript knowledge, but I don't see a way to specify a default for "TlvField". Is there a way?

Reply:
You are basically right ... there will come a fallback value soon but this is for a different behaviour, so you are right ... :-) leave as comments "// default: ..." please